website/docs/reference/commands/cmd-docs.md

```diff
@@ -10,11 +10,12 @@ id: "cmd-docs"
 
 The command is responsible for generating your project's documentation website by
 
-1. Copying the website `index.html` file into the `target/` directory 
+1. Copying the website `index.html` file into the `target/` directory
 2. Compiling the resources in your project, so that their `compiled_code` will be included in [`manifest.json`](/reference/artifacts/manifest-json)
```

Comment:
```suggestion
2. Compiling the resources in your project, so that their `compiled_code` will be included in [`manifest.json`](/reference/artifacts/manifest-json).
```